Makefile

```diff
@@ -0,0 +1,19 @@
+CRYSTAL_BIN ?= $(shell which crystal)
+PREFIX ?= /usr/local
+SHARD_BIN ?= ../../bin
+
+build:
+	$(CRYSTAL_BIN) build --no-debug -o bin/neph src/bin/neph_bin.cr $(CRFLAGS)
+clean:
+	rm -f ./bin/neph
+install: build
+	mkdir -p $(PREFIX)/bin
```

Comment:
The `mkdir` command can fail if target directory exists. I recommend using `install`.

Reply:
not if **-p** is specified

Reply:
Oops, I have forget `-p`. You're right.